python/setup.py

```diff
@@ -39,6 +39,8 @@
 from distutils.util import strtobool
 from distutils import sysconfig
 
+from setuptools import setup
```

Comment:
That import already exists above.

Reply:
Not sure why I missed the existing and added this import. I'll get that corrected and also setup my linter to find duplicate imports. Thanks for catching.